core/src/avm2/globals/flash/desktop/ClipboardFormats.as

```diff
@@ -8,6 +8,18 @@ package flash.desktop
 
     public class ClipboardFormats
     {
+        // Image data (AIR only).
+        [API(661)]
```

Comment:
The way most other API-versioning metadata is, this would be `[API("661")]`, with quotes.

Reply:
Oh, you're right. Fixed.
Speaking of, maybe we should fix it instead, and do it without quotes everywhere? Just for simplicity. And that would make it "feel" more like an integer, which I think it is. Would that work?